waku/node/peer_manager/peer_manager.nim

```diff
@@ -122,13 +122,14 @@ proc addPeer*(pm: PeerManager, remotePeerInfo: RemotePeerInfo, origin = UnknownO
   ## Adds peer to manager for the specified protocol
 
   if remotePeerInfo.peerId == pm.switch.peerInfo.peerId:
-    # Do not attempt to manage our unmanageable self
+    trace "skipping to manage our unmanageable self"
     return
 
   if pm.peerStore[AddressBook][remotePeerInfo.peerId] == remotePeerInfo.addrs and
       pm.peerStore[KeyBook][remotePeerInfo.peerId] == remotePeerInfo.publicKey and
       pm.peerStore[ENRBook][remotePeerInfo.peerId].raw.len > 0:
-    # Peer already managed and ENR info is already saved
+    trace "peer already managed and ENR info is already saved",
```

Comment:
Not related to this PR...but what if ENR has been updated, in that case do we update the ENR stored and peers capabilities locally?

Reply:
> Not related to this PR...but what if ENR has been updated, in that case do we update the ENR stored and peers capabilities locally?

The local peer store only gets new entries when a new connections is established

Reply:
Ok, where is the ENR record stored for a peer? Is it part of peerStore or somewhere else?